Sources/OpenAPI/.create-api.yml

```diff
@@ -1,46 +1,21 @@
-# Modifier for all generated declarations
-access: public
-# Add @available(*, deprecated) attributed
-isAddingDeprecations: true
-# Available values: ["spaces", "tabs"]
+generate: [entities, paths, enums]
+module: AppStoreConnect_Swift_SDK
```

Comment:
Rather than passing the module name to `--package` or `--module`, we set it here now 👍